CMakeLists.txt

```diff
@@ -161,32 +163,34 @@ ELSE (GIT_BINARY)
 	ENDIF (NOT GIT_FOUND)
 ENDIF (GIT_BINARY)
 
-IF (PREFIX)
+# PREFIX has no sence in case of win32
```

Comment:
Thas copy-paste.
27.07.2015 13:05 пользователь "BevapDin" <notifications@github.com> написал:

> In CMakeLists.txt
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/13047#discussion_r35521454>
> :
>
> > @@ -161,32 +163,34 @@ ELSE (GIT_BINARY)
> >  	ENDIF (NOT GIT_FOUND)
> >  ENDIF (GIT_BINARY)
> >
> > -IF (PREFIX)
> > +# PREFIX has no sence in case of win32
>
> sence? Did you meant "sense", as "makes no sense"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CleverRaven/Cataclysm-DDA/pull/13047/files#r35521454>.
>